crates/db_schema/src/impls/local_user.rs

```diff
@@ -1,12 +1,6 @@
 use crate::{
   newtypes::{DbUrl, LocalUserId, PersonId},
-  schema::local_user::dsl::{
```

Comment:
I don't like importing these dsl columns, much cleaner to just do `local_user::COLUMN`